### ISSUE TITLE
Hitting "Enter" when have Tags tab open copies the abstract multiple times as a Tag

### ISSUE THREAD

**Issue by oo7bnoremac** [wshanks/Zutilo#87]
Prior to adding this add-on, you could type a tag name then hit enter and it would create a new tag box for you to type into below. As you can see from the images below, I hit enter after typing Sarcopenia and the other image is what occurs now. 

<img width="336" alt="screen shot 2018-07-03 at 1 50 27 pm" src="https://user-images.githubusercontent.com/40805634/42238992-37640baa-7ec8-11e8-84c8-e4b38d17ab41.png">

<img width="338" alt="screen shot 2018-07-03 at 1 50 36 pm" src="https://user-images.githubusercontent.com/40805634/42238993-377d8922-7ec8-11e8-99b7-63115e65047c.png">

**Comment by oo7bnoremac** [comment]
If I enter a ISBN, DOI, PMID, etc. then that number gets pasted as a Tag for the last added document.

**Comment by oo7bnoremac** [comment]
I think the issue lies here. I just recently redownloaded Zutilo and this is still like this. Paste Tags does not appear as <disabled>…even after hitting “Disable” and then hit “Apply” it still shows as blank there. Since I am seeing an issue with copying and pasting tags I believe your issue is here.

**Comment by oo7bnoremac** [comment]
Ok it is possibly a little more complex than the last email I just sent. I typed in “Nutrition” as a tag, hit enter twice so as to type in a new tag and then it automatically pasted what I had saved to my clipboard which was an abstract. At this point I have all possible shortcuts in Zutilo set to <disabled>

**Comment by oo7bnoremac** [comment]
Problem #3

The tag with the doc.org link is what I pasted into the button to add items by identifier then hit enter. That doi.org <http://doi.org/> is for the top article labeled Prevalence and Co-Occurence…  After hitting enter, it brings up that document and adds the pasted link as a tag to whatever article i have selected already.

**Comment by oo7bnoremac** [comment]
I would really like to solve this issue as I believe your add on can do wonders for my ability to enter in over 3,000 more articles for my boss. If best, I can speak on the phone, FaceTime or any other means to speed this up. I appreciate your help.

**Comment by willsALMANJ** [comment]
A rogue shortcut setting does seem like a possibility. Things I would try:
1. Disable Zutilo in the Add-ons Manager and confirm that the weird behavior goes away.
2. Open the Config Editor from the Advanced tab of the Preferences window, enter "extensions.zutilo" into the search field, and then reset all the bold preferences to the default values (right click them and choose Reset).
3. Try installing Zutilo into a new Zotero profile like I suggested above and make sure the behavior is not seen there.
I have very little free time right now, so scheduling a real time meeting will be tough. Let's see how the suggestions above work out.

**Comment by oo7bnoremac** [comment]
1. The weird behavior goes away once Zutilo is disabled completely. 

2. Here is what I see before performing the resets.
  

3. After resetting all of those items, I turn Zutilo back on and go back into the preferences of Zutilo and go to shortcuts. When I click on Paste Tags and try to change the shortcut to “Option + Command + V” it becomes empty again as seen in the image below: (Note: Even if i now click disable for the shortcut for Paste Tags, it stays empty.)



Also below, you can see that the shortcut I entered for Copy Tags appears in the list unlike what happens with Paste Tags:

**Comment by willsALMANJ** [comment]
It looks like your images didn't come through.

After you reset all of Zutilo's settings, does Zotero behave normally? Is it only after setting the Paste Tags shortcut that the strange behavior begins? I am curious to see what shows up for the Paste Tags shortcut in the config editor. Perhaps there is a problem with using option and command together in a shortcut?

**Comment by iansethmcphail** [comment]
Like @oo7bnoremac I have the same problem.

Upon hitting enter after typing a new tag name, multiple tags created from (what seems to be) the clipboard are generated. Problem is solved when Zutilo is disabled. I have not tried with another Zotero account (yet).

> Paste Tags does not appear as < disabled > …even after hitting “Disable” and then hit “Apply” it still shows as blank there. 

This is the case in my situation as well. 

Per @willsALMANJ 's instructions, I rest all of Zutilo's settings in the config editor. Which does seem to solve the issue. However, like @oo7bnoremac , the Paste Tags shortcut in the Zutilo preference menu turns blank again once I attempt to change it to “Option + Command + V." 

> After resetting all of those items, I turn Zutilo back on and go back into the preferences of Zutilo and go to shortcuts. When I click on Paste Tags and try to change the shortcut to “Option + Command + V” it becomes empty again as seen in the image below: (Note: Even if i now click disable for the shortcut for Paste Tags, it stays empty.)

This ultimately leads to the core issue recreating itself again (multiple tags generated from clipboard after hitting enter when inputting new tag name).
I reset the Zutilo preferences once more and again, problem goes away.

> I am curious to see what shows up for the Paste Tags shortcut in the config editor. Perhaps there is a problem with using option and command together in a shortcut?

I think you're onto something here. 
I tried "Control + V" instead of "Option + Command + V" and everything works fine. The shortcut appears in the Zutilo menu as it is supposed to and it is possible to disable it properly. 

Could it indeed be the combination of option and command that is causing the strange behaviour?
Just now, I also noticed that another issue I was having (unable to Edit item info with the shortcut " Option + Command + I") works absolutely fine with Control + I.

Not being able to use Option in shortcuts would be a bit disappointing but if that's truly our problem then I am more than happy to find alternative shortcut options that work.

Cheers!

**Comment by qqobb** [comment]
> I tried "Control + V" instead of "Option + Command + V" and everything works fine.

Thanks for figuring out this workaround. Please let us know if you find a better solution.

This issue seems to require troubleshooting [Mac keyboard shortcuts](https://support.apple.com/en-us/HT201236). There's probably not much Zutilo can do about it, so I'll close this issue.

**Comment by qqobb** [comment]
BTW, this might be related to #136.
